cmake/DARTConfig.cmake.in

```diff
@@ -19,6 +19,23 @@ set(DART_LIBRARY_DIRS "@CMAKE_INSTALL_PREFIX@/lib" "@Boost_LIBRARY_DIRS@")
 
 include("${CURRENT_DIR}/DARTTargets.cmake")
 
-set(DART_LIBRARIES dart)
+find_library(dart_location NAMES "dart" PATHS "@CMAKE_INSTALL_PREFIX@/lib")

```

Comment:
> This wouldn't work if DART is built in debug mode since the library name would contain appendix "d" 

This is a good point. After looking into this a bit, it seems like the standard way to handle this would be to have two versions of the find_package macro: one for the release version and one for the debug version. That said, I'd be open to suggestions for other ways to have ``find_package`` distinguish between release and debug libraries.

> Also, I haven't seen quotation mark is used for library name in find_library. Do we need it?

It looks like quotes in cmake are only really used to ensure that an argument with spaces in it does not get split into multiple arguments. That being the case, the quotes are probably worthless here.